README.md

```diff
@@ -1,72 +1,69 @@
 # Circuits for zkEVM
 
-This is the zkEVM circuits used in Scroll Mainnet. It was forked from PSE zkevm-circuits and added a lot of new features later:
+This is the zkEVM circuits used in Morph Mainnet. It was forked from Scroll zkevm-circuits and a lot of new features were added later and in the future:
```

Comment:
It was forked from Scroll zkevm-circuits and a lot of new features were added later and in the future -> It was forked from Scroll zkevm-circuits, with many new features added and more planned for the future.